style.css

```diff
@@ -0,0 +1,51 @@
+header {
+    text-align: center;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+span {
+    color: blue;
+    font-style: italic;
+}
+
+nav{
+    text-align: center;
+    border: 2px solid #000;
+    box-shadow: 5px 5px 10px rgba(0, 0, 0, 0.5);
+}
+
+main {
```

Comment:
nitpick: see if this style can be condensed with the style below it. and, see if there are any other places where other repeated styles can be condensed.